db/migrate/20170616152705_set_codelocation_status_to_nil.rb

```diff
@@ -0,0 +1,93 @@
+class SetCodelocationStatusToNil < ActiveRecord::Migration
```

Comment:
We used to create a migration file only to modify the schema. As this file deals with updating the existing values in the table, we could move this file under scripts folder.